### PR TITLE
feature:open and close legend window

### LINKED ISSUE #6: Open and hide legend window
Make function, which open and close window with legend.

### PR DESCRIPTION
**Pull request by rinatGIS**

### CODE REVIEW

File: src/components/Legend.vue [hunk 1]
```diff
@@ -31,19 +31,34 @@
...
         </b-row> -->
-      Легенда
+    <v-container fluid>
+      <v-row align="center">
+        <v-col cols="10" >
+          Легенда
+        </v-col>
+        <v-col cols="1">
+          <v-btn icon @click="setLegendVisibility(false)">
+            <v-icon size="24px"> mdi-close </v-icon>
```
Comment (andrewGIS) [inline src/components/Legend.vue:18]: Почему размер 24?

File: src/components/Legend.vue [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <template>
   <!-- <l-control position="bottomright" > -->
   <v-snackbar
-    v-model="snackbar"
```
Comment (andrewGIS) [inline src/components/Legend.vue:4]: Why is it deleted?

File: src/components/Legend.vue (hunk 1 above)
Comment (rinatGIS) [reply src/components/Legend.vue:18]: Icon resolution

File: src/components/Legend.vue (hunk 2 above)
Comment (rinatGIS) [reply src/components/Legend.vue:4]: It worked without this property.

**Review by andrewGIS — CHANGES_REQUESTED** [review]
Not everything is okey

**Review by andrewGIS — APPROVED** [review]

**Pull request merged**